src/core_plugins/metrics/public/kbn_vis_types/editor_controller.js

```diff
@@ -15,6 +15,7 @@ function ReactEditorControllerProvider(Private, localStorage, config) {
 
       const autoApply = localStorage.get(AUTO_APPLY_KEY);
       vis.autoApply = autoApply != null ? autoApply : true;
+      vis.initialized = true;
```

Comment:
I thought the only time this would be needed to be manually set if the Visualization had an init function?  Otherwise in visualization.js it should fall into the second path here:
```
      if (visualization.init) {
        visualization.init().then(() => { $scope.vis.initialized = true; });
      } else {
        $scope.vis.initialized = true;
      }
```

Reply:
tsvb is not using <visualization> to display the visualizations when in editor mode (it has its own editor, which does things in different manners). so its editor needs to tell when visualization is ready to receive data